contracts/HATVaults.sol

```diff
@@ -105,5 +105,6 @@ contract HATVaults is Claim, Deposit, Params, Pool, Swap, Getters, Withdraw {
             claimFee: 0
         });
         setRewardController(_rewardController);
+        arbitrator = owner();
```

Comment:
we may want to set with an argument in the intialiazer

Reply:
I agree, let's do it